tidy3d/components/data/transformations.py

```diff
@@ -0,0 +1,156 @@
+"""Coordinate system and field transformations."""
```

Comment:
This is probably ok, although I feel like maybe they could go, say, in the `Geometry` class. And then we could call them from the `DiffractionMonitor.car_2_sph` for example, and in other places (like they could be useful for arrow display in geometry). Just a thought. does it make sense with the rest of the workflow?

Reply:
Yeah that makes a lot of sense